content/whatsapp-signal.md

```diff
@@ -3,6 +3,34 @@ title: Switch from WhatsApp to Signal
 description: Switch from WhatsApp to Signal
 ---
 
+
+!()[whatsanalyze.com/whatsapp-vs-signal.jpg]
```

Comment:
![Not working](https://user-images.githubusercontent.com/23440847/111487090-59d54700-8738-11eb-8748-43c37bea9235.png)